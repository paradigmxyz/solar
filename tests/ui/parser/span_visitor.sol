//@compile-flags: -Zspan-visitor

contract SpanVisitorTest {
    uint256 x;
    
    function foo() public {
        x = 42;
    }
}