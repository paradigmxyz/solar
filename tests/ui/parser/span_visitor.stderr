note: span #1
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL | / contract SpanVisitorTest {
LL | |     uint256 x;
...  |
LL | |     }
LL | | }
   | |_-
   |

note: span #2
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL | contract SpanVisitorTest {
   |          ---------------
   |

note: span #3
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     uint256 x;
   |     ----------
   |

note: span #4
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     uint256 x;
   |     ----------
   |

note: span #5
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     uint256 x;
   |     -------
   |

note: span #6
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     uint256 x;
   |             -
   |

note: span #7
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL | /     function foo() public {
LL | |         x = 42;
LL | |     }
   | |_____-
   |

note: span #8
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     function foo() public {
   |     ---------------------
   |

note: span #9
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     function foo() public {
   |              ---
   |

note: span #10
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     function foo() public {
   |                 --
   |

note: span #11
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |     function foo() public {
   |                    ------
   |

note: span #12
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |       function foo() public {
   |  ___________________________-
LL | |         x = 42;
LL | |     }
   | |_____-
   |

note: span #13
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |         x = 42;
   |         -------
   |

note: span #14
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |         x = 42;
   |         ------
   |

note: span #15
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |         x = 42;
   |         -
   |

note: span #16
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |         x = 42;
   |         -
   |

note: span #17
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |         x = 42;
   |             --
   |

note: span #18
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |         x = 42;
   |             --
   |

note: span #19
  --> ROOT/tests/ui/parser/span_visitor.sol:LL:CC
   |
LL |       function foo() public {
   |  ___________________________-
LL | |         x = 42;
LL | |     }
   | |_____-
   |

